src/shared/README.txt

```diff
@@ -0,0 +1,22 @@
+The "shared" directory contains the common code between the runtime and diagnostics repros.
+
+It is also shared by the dbgshim and SOS components.
+
+Updated last from the runtime repo commit hash: 442a42147ef23c3b9742abcd8b997e8f472af68a
+
+runtime/src/coreclr/inc -> diagnostics/src/shared/inc
```

Comment:
Nice rules - something like this sounds somewhat scriptable (granted not as easy because of the trimming)